app_dart/pubspec.yaml

```diff
@@ -25,7 +25,7 @@ dependencies:
   googleapis_auth: ^1.3.1
   gql: ^0.14.0
   graphql: ^5.1.1
-  grpc: ^3.0.2
+  grpc: ">=3.0.2 <3.1.0"
```

Comment:
Can you add a link to https://github.com/dart-lang/appengine/issues/154

Reply:
done!